docs/Installing.asciidoc

```diff
@@ -23,26 +23,33 @@ Installation
 ------------
 [id="installation"]
 
-The easiest way to install openQA is from packages. You can find them in OBS in
-the https://build.opensuse.org/project/show/devel:openQA[openQA] repository or
+The easiest way to install openQA is from packages. You can find openSUSE
+packages in OBS in the
+https://build.opensuse.org/project/show/devel:openQA[openQA] repository or
```

Comment:
I guess you meant to add `:stable` suffix (https://build.opensuse.org/project/show/devel:openQA:stable) else the URL is same as lower for openQA:devel